### PR TITLE
Added error string for process operations

### PR DESCRIPTION
**Pull request by rajasec**
Added error strings when there is no process operations.

Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

File: libcontainer/error.go [hunk 1]
```diff
@@ -46,6 +46,8 @@ func (c ErrorCode) String() string {
 		return "Container is not running"
 	case ConsoleExists:
 		return "Console exists for process"
+	case ProcessNotExecuted:
+		return "No process operations"
```
Comment (hqhq) [inline libcontainer/error.go]: The name and message seems not well matched, can we change `ProcessNotExecuted` to something like `NoProcessOps`? The name `ProcessNotExecuted` seems has more meaning than on ops.

Comment (rajasec) [inline libcontainer/error.go]: @hqhq 
I've fixed and committed. Please review

**Comment by crosbymichael** [comment]
@rajasec why did you not keep `ProcessNotExecuted` and just added the string for it?

**Comment by rajasec** [comment]
@crosbymichael 
Whenever ProcessNotExecuted error occurs, string printed will be "uknown error" so I have added the error string to show there is no process operations.
From the review comments I've changed the error name to NoProcessOps instead of ProcessNotExecuted
Let me know if you are OK.

**Comment by hqhq** [comment]
Consider what we have for errors:
```
   9 const (
 10         // Factory errors
 11         IdInUse ErrorCode = iota
 12         InvalidIdFormat
 13
 14         // Container errors
 15         ContainerNotExists
 16         ContainerPaused
 17         ContainerNotStopped
 18         ContainerNotRunning
 19
 20         // Process errors
 21         ProcessNotExecuted
 22
 23         // Common errors
 24         ConfigInvalid
 25         ConsoleExists
 26         SystemError
 27 )
```
For process errors, I think all kinds of process errors could cause process not executed, so we may need several specific errors to show more detailed information like other errors, so `NoProcessOps` could be a start. @crosbymichael WDYT?

**Comment by rajasec** [comment]
@crosbymichael 
Please let me know your comments ?

**Comment by crosbymichael** [comment]
Ok, sounds good.  @rajasec can you rebase?

**Comment by rajasec** [comment]
@crosbymichael 
I'm done with rebase.

**Comment by hqhq** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**